data.ts

```diff
@@ -33,8 +33,8 @@ export const languages: Record<string, Language> = {
     name: 'Rust',
     color: '#814C21',
   },
-  elixir: {
-    name: 'Elixir',
+  erlang_elixir: {
```

Comment:
```suggestion
  erlangElixir: {
```

Nit: Use camelCase here https://github.com/airbnb/javascript#naming--camelCase